stuff/pipelines/client_contest.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python3
+
+'''Processing contest messages.'''
+
+import argparse
+import logging
+import os
+import sys
+import json
+import MySQLdb
+import MySQLdb.cursors
+import pika
+from verification_code import generate_code
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
+import lib.db   # pylint: disable=wrong-import-position
+import lib.logs  # pylint: disable=wrong-import-position
+
+
+def receive_contest_messages(
```

Comment:
aquí falta decir que es para certificados:

```suggestion
def certificate_contests_receive_messages(
```

las palabras van en orden de general a específico.

Reply:
Listo